agent/wptdriver/wpt_test.cc

```diff
@@ -286,8 +288,13 @@ bool WptTest::Load(CString& test) {
         else if (!key.CompareNoCase(_T("addCmdLine")))
           _browser_additional_command_line = value;
         else if (!key.CompareNoCase(_T("continuousVideo")) &&
-                 _ttoi(value.Trim()))
+          _ttoi(value.Trim()))
           _continuous_video = true;
+        else if (!key.CompareNoCase(_T("timeout"))) {
+          _overall_timeout_in_ms = _ttoi(value.Trim()) * 1000;

```

Comment:
Why a new property and not just use _test_timeout?  It is initialized to a default value but that doesn't mean it couldn't/shouldn't be overridden.